classes/article/PublishedArticleDAO.inc.php

```diff
@@ -154,12 +154,15 @@ function getPublishedArticlesByJournalId($journalId = null, $rangeInfo = null, $
 				' . $this->getFetchColumns() . '
 			FROM	published_submissions ps
 				LEFT JOIN submissions s ON ps.submission_id = s.submission_id
+				LEFT JOIN submission_settings ss ON ss.submission_id = ps.submission_id
 				LEFT JOIN issues i ON ps.issue_id = i.issue_id
 				' . $this->getFetchJoins() . '
 			WHERE 	i.published = 1
 				' . ($journalId?'AND s.context_id = ?':'') . '
 				AND s.status <> ' . STATUS_DECLINED . '
-			ORDER BY ps.date_published '. ($reverse?'DESC':'ASC'),
+				AND ss.setting_name = "datePublished"
```

Comment:
@lilients, standard SQL syntax is for strings to be quoted with `'` -- MySQL is OK with `"` as well, but I think PostgreSQL isn't.